testsuite/tests/ui/conftest.py

```diff
@@ -322,6 +321,29 @@ def get_resultsdir_path(node):
     return path
 
 
+def fullpage_screenshot(driver, file_path):
+    """
+        A full-page screenshot function. It scroll the website and screenshots it.
+        - Creates multiple files
+        - Screenshots are made only vertically (on Y axis)
+    """
+    # Removal of the height: 100% style, that disables scroll.
```

Comment:
As @dhlavac mentioned if this block of code throws some error the whole test will fail even the test run passed. Add try block to prevent this from happening.

Reply:
The question is then ... do we want to let the tester know that screenshots failed as well ?